llvm/test/Transforms/GlobalOpt/cleanup-pointer-root-loads.ll

```diff
@@ -0,0 +1,42 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
```

Comment:
please add new test coverage in a separate PR (i.e. precommit them, so that here we only see the functional diff)

Reply:
https://github.com/llvm/llvm-project/pull/87443